.github/pull_request_template.md

```diff
@@ -6,12 +6,15 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
-
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+Please remove the segment/s on either side of the | symbol as appropriate, and add any relevant link/s to the end of the line.
+Use # to link to relevant PRs if the provided content is not part the present PR.
+
+- [ ] **changelog** provided, | not required because: 
```

Comment:
What does that `,` at the end of `provided,` mean? Are we expecting some more information there if that branch is  taken?

Reply:
That was a accidental and has been removed.